### PR TITLE
docs: added difference between get_ commands and fetch_ commands in FAQ

### PR DESCRIPTION
**Pull request by Blue-Robin-Taken**
Added the difference between get_ commands and fetch_ commands

## Information

- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [ ] I have updated the changelog to include these changes.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -109,6 +109,18 @@ Putting both of these pieces of info together, you get the following: ::
     activity = discord.Activity(name='my activity', type=discord.ActivityType.watching)
     client = discord.Client(activity=activity)
 
+What is the difference between get_ commands and fetch_ commands?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+get_ commands retrieve from the cache and won't always raise if not found. fetch_ commands always make an API request and if not found, raise. Example: ::
```
Comment (Lulalaby) [inline docs/faq.rst:114]: Bad unprofessional wording

**Review by Lulalaby — CHANGES_REQUESTED** [review]

File: docs/faq.rst (hunk 1 above)
Comment (Blue-Robin-Taken) [reply docs/faq.rst:114]: What would be "professional"?

**Review by Lulalaby — DISMISSED** [review]

File: docs/faq.rst [hunk 2]
```diff
@@ -109,6 +109,18 @@ Putting both of these pieces of info together, you get the following: ::
     activity = discord.Activity(name='my activity', type=discord.ActivityType.watching)
     client = discord.Client(activity=activity)
 
+What is the difference between get_ methods and fetch_ methods?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"Get" methods retrieve objects or data from the cache (general store of data) and can return ``None`` if not found (this doesn't raise an error). "Fetch" methods force an API request and will raise an error if not found. Example: ::
```
Comment (JustaSqu1d) [inline docs/faq.rst:114]: or
```suggestion
"Get" methods retrieve objects or data from the cache (general store of data) and can return ``None`` if not found. This doesn't raise an error. "Fetch" methods force an API request and will raise an error if not found. Example: ::
```

[the remark this answers is not in the document]
Comment (JustaSqu1d) [reply docs/faq.rst:114]: I don't see why this is necessary.

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]

**Review by ChickenDevs — DISMISSED** [review]

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]

**Comment by sus2790** [comment]
How about `get_or_fetch`?

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]
> I think using a simpler example like bot.fetch_user vs bot.get_user would be easier to understand.

My previous review still applies.

**Comment by Lulalaby** [comment]
I think this change leads to nowhere.
I'm still having the opinion that it's obvious.
We shouldn't deal with user stupidness.

**Comment by JustaSqu1d** [comment]
> I think this change leads to nowhere. I'm still having the opinion that it's obvious. We shouldn't deal with user stupidness.

Saying that you need `members` intent to do this in another method instead of the function itself, for example, is anything, but obvious.

**Comment by Lulalaby** [comment]
Then please create a solid pull request yourself with a correct faq about it or better document it in the docstring

**Comment by Blue-Robin-Taken** [comment]
@Lulalaby What exactly do you want? 

> We shouldn't deal with user stupidness.

Please be professional. I know it's an online repo, but I'm a student. I'm still learning.

What would be a "solid" PR, and can't I edit and fix the PR here?

**Comment by Lulalaby** [comment]
"user stupidness" wasn't meant as in your pr.
I meant it as in we shouldn't faq tf outta everything.

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]
> > I think using a simpler example like bot.fetch_user vs bot.get_user would be easier to understand.
> 
> My previous review still applies.

You can't just re-request a review without acknowledging my previous one.

**Comment by plun1331** [comment]
my thoughts on this: i personally haven't seen this question frequently asked

**Comment by Lulalaby** [comment]
Yeah exactly that and that's why I've closed it in the first place. But squid reopened it

**Comment by Blue-Robin-Taken** [comment]
> > > I think using a simpler example like bot.fetch_user vs bot.get_user would be easier to understand.
> > 
> > 
> > My previous review still applies.
> 
> You can't just re-request a review without acknowledging my previous one.

I thought I did but I don't have much github experience... Sorry about that. I'll try to fix it

**Comment by Lulalaby** [comment]
simple: no where

**Comment by plun1331** [comment]
> > my thoughts on this: i personally haven't seen this question frequently asked
> 
> Although it's not frequently asked, it's gotta be listed somewhere when it is asked. 
> Where else should it be listed then? @plun1331 

In the entire time I've been with pycord I've only observed it asked 3-5 times, and the answer is fairly simple to explain each time: get_x gets from cache, fetch_x fetches from discord

If we had to make an entry for every question asked about the library, this FAQ page would be extremely long

**Comment by Lulalaby** [comment]
And so I'm closing it.

**Pull request closed without merging**